src/transformers/training_args.py

```diff
@@ -1261,6 +1264,12 @@ class TrainingArguments:
     include_inputs_for_metrics: bool = field(
         default=False, metadata={"help": "Whether or not the inputs will be passed to the `compute_metrics` function."}
     )
+    eval_do_concat_batches: bool = field(
+        default=True,
+        metadata={
+            "help": "Whether or not tensors in nested objects in batches should be recursively concatenated between batches."
```

Comment:
Let's keep this to the same docstring as above

Reply:
thanks, fixed